application_processor/src/application_processor.c

```diff
@@ -38,7 +38,8 @@
 
 // Includes from containerized build
 #include "ectf_params.h"
-#include "global_secrets.h"
+#include "ap_secrets.h"
+#include "component.c"
```

Comment:
`component.c: No such file or directory`
Can we comment this line out? It doesn't seem necessary and is preventing the code from compiling.